### ISSUE TITLE
Gallery TODOs

### ISSUE THREAD

**Issue by yotan-dev** [yotan-dev/mad-island-mods#4]
- [ ] Make other scenes show/work in gallery
- [ ] Properly clean up current scene before starting a new one
- [ ] Somehow allow you to customize the characters in the scene -- idea: keep track of the last loaded file NPCs so you can use them in gallery; idea 2: make special gallery characters that can be customized in a surgery table-like UI

**Comment by NamelessButler** [comment]
For customizing character it could work if you mean changing NPC appearances.
But changing actual npcs it won't as they have the textures atlas and animation in the spine skelleton.

**Comment by yotan-dev** [comment]
yeah, I was thinking about making presets like you make "Native 1 -> hair 1, ..." and then for scenes with native, you can use this one.

**Comment by alreadydead0000** [comment]
Grapple fight is not working. Grapple skill tree is useless now

**Comment by yotan-dev** [comment]
I think I fixed it on https://github.com/yotan-dev/mad-island-mods/releases/tag/v2025.03.11%2B1 - thanks!
I ended up not seeing it here
